standard.md

```diff
@@ -6,18 +6,17 @@
 
 Indicator |	Requirement
 --- | ---
-**1. Relevance to Sustainable Development Goals** |	All projects must indicate the SDG(s) that they are relevant to and provide supporting links/documentation to support this relevance. 
+**1. Relevance to sustainable development goals** |	All projects must indicate the SDG(s) that they are relevant to and provide supporting links/documentation to support this relevance. 
```

Comment:
Hey @LawrenceHecht, it would be good to keep this capitalized. This is referring to the UN Sustainable Development Goals:

https://www.undp.org/content/undp/en/home/sustainable-development-goals.html

Perhaps we could add this as a URL there too for clarity.

Reply:
I think it is OK to keep it lowercase, because SDG is mentioned in the action definition of the requirement. I'm going to make another small suggested edit. that will add the link you mentioned.

Reply:
@jwflory or maybe I won't actually make that suggested change. I think adding the URL to SDG(s) in the req definition is sufficient.

Reply:
Thanks @jwflory and @LawrenceHecht for your input on this. As per our [governance] guidelines, I'm splitting the many proposed changes of this PR, into individual changes to make them easier to triage. Thus, I'm moving this particular one to #26, and resolving this conversation.